### PR TITLE
Improvements to 'include' macro documentation.

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

**Comment by frewsxcv** [comment]
I'm open to a better example if someone has a better idea.

File: src/libstd/macros.rs [hunk 1]
```diff
@@ -470,11 +470,26 @@ pub mod builtin {
...
+    ///
+    /// ```ignore
+    /// fn main() {
+    ///     let my_str = include!("my_str.in");
+    ///     println!("{}", my_str);
     /// }
     /// ```
+    ///
+    /// Compiling 'main.rs' and running the resulting binary will print "Hello
+    /// World".
```
Comment (durka) [inline src/libstd/macros.rs]: Pedantically, I guess it will actually print "Hello World!"

File: src/libstd/macros.rs [hunk 2]
```diff
@@ -458,8 +458,8 @@ pub mod builtin {
 
     /// Parse a file as an expression or an item according to the context.
     ///
-    /// The file is located relative to the current file. (similarly to how
-    /// modules are found)
+    /// The file is located relative to the current file (similarly to how
+    /// modules are found).
     ///
     /// Using this macro is often a bad idea, because if the file is
     /// parsed as an expression, it is going to be placed in the
```
Comment (durka) [inline src/libstd/macros.rs]: Might as well fix the small typo on the next line (should be "unhygienically")

**Review by durka — APPROVED** [review]
I don't know if it's worth including a warning about #35560? (i.e. don't `include!` a file with more than one statement in it)

**Comment by frewsxcv** [comment]
@durka Thanks for the feedback! Comments have been addressed.

**Comment by GuillaumeGomez** [comment]
Thanks for the examples and the fixes!

@bors: r+ rollup

**Pull request merged**